src/displayBackend/DisplayBackend.cpp

```diff
@@ -128,7 +128,13 @@ void DisplayFrontendHandler::createConnector(const string& conPath,
 
 	auto id = getXenStore().readString(conPath + XENDISPL_FIELD_UNIQUE_ID);
 
-	auto connector = mDisplay->createConnector(id);
+	auto resolution = getXenStore().readString(conPath +
+											   XENDISPL_FIELD_RESOLUTION);
+	auto pos = resolution.find(XENDISPL_RESOLUTION_SEPARATOR);
+	auto width = stoi(resolution.substr(0, pos));
+	auto height = stoi(resolution.substr(++pos));
```

Comment:
Do we have guaranies that stoi won't fail ? (Do we need handle exception here ?)

Reply:
the relevant exception is properly handled at the upper layer. you can see the above code can also raise exceptions